sbol2/constants.py

```diff
@@ -291,3 +294,8 @@
 SYSBIO_SAMPLE_ROSTER = SYSBIO_URI + "#SampleRoster"
 
 IGEM_URI = "http://wiki.synbiohub.org/wiki/Terms/igem"
+
+OM_NS = rdflib.Namespace(OM_URI)
+OM_MEASURE: str = OM_NS.Measure.toPython()
+OM_HAS_UNIT: str = OM_NS.hasUnit.toPython()
+OM_HAS_NUMERICAL_VALUE: str = OM_NS.hasNumericalValue.toPython()
```

Comment:
This is an interesting pattern I've not seen before.  Whats the advantage of doing it this way rather than simple assignment of a string value?   How is it possible you are calling the `toPython` method on `OM_NS.Measure` and `OM_NS.hasUnit` when these don't appear to be pre-defined anywhere?  So. many. questions!

Reply:
I thought we could try this out and see if we like the pattern or not. I don't know if it is beneficial.

An `rdflib.Namespace` will automatically create a URI for any attribute dynamically, so `OM_NS.hasUnit` generates a URI that is the concatenation of `OM_NS` and `hasUnit`. In that sense it is no different from string operations. It does make it a little more explicit that we're appending to a namespace. I don't know if that is a worthwhile distinction to make.

Reply:
Interesting.  I think this `rdflib.Namespace` behavior should be leveraged in SBOL3 for generating URIs instead of hard-coding constants.